tutorials/asr/asr_adapters/ASR_with_Adapters.ipynb

```diff
@@ -1,25 +1,4 @@
 {
```

Comment:
Why such a huge change? Please revert this and manually make edits in colab, then save the modified version. That would reduce the changes coming from Jupiter

Reply:
not sure...Editing jupyter notebooks in github is alway a pain...let me refine it.